index.css

```diff
@@ -0,0 +1,109 @@
+@import url('https://fonts.googleapis.com/css2?family=Lacquer&family=Poppins&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+}
+
+body {
+  background-image: url('./img/background_image.png');
+}
```

Comment:
- [ ] Great job implementing the changes suggested by the previous reviewer. One of the objectives of this project is to understand how to parse a design template precisely. Let's work on the background image as suggested by the previous reviewer and mentioned in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md) to match the Figma design. 🤝️

**Project Requirements**
![Screenshot from 2023-02-16 19-55-33](https://user-images.githubusercontent.com/17243325/219436235-55f98ed2-6bc2-4283-9596-5539e6921ff2.png)


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/219435846-f0b1f768-2c54-4f79-8c38-9261115eb6bb.png" width="300">|<img src="https://user-images.githubusercontent.com/17243325/219435857-7bc3c6a9-58fc-4ddd-a678-4a36138629cf.png" width="300">|

- [ ] You used the correct font size but the style seems a bit different because the bold type was not imported from google fonts. You can try the following for better results. 😎️

```
@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap');
```

Reply:
Hello @amejid,

Thank you for the explanation. Regarding the background image, is there any suggestion you have for me?, I've tried different background properties but none seems to work as expected.

Thanks in advance!

Reply:
Hi @J-C-S-V, I think the image you currently have inside the repo is supposed to be used for the desktop version. Please grab the background image for the mobile version from the Figma template.